peachjam/views/documents.py

```diff
@@ -9,6 +11,96 @@
 from peachjam.registry import registry
 
 
+class RedirectResolver:
+
+    RESOLVER_MAPPINGS = {
+        "africanlii": {
+            "country_code": "aa",
+            "domain": "agp.africanlii.org",
+        },
+        "ghalii": {
+            "country_code": "gh",
+            "domain": "new.ghalii.org",
+        },
+        "lawlibrary": {
+            "country_code": "za",
+            "domain": "lawlibrary.org.za",
+        },
+        "lesotholii": {
+            "country_code": "ls",
+            "domain": "new.lesotholii.org",
+        },
+        "malawilii": {
+            "country_code": "mw",
+            "domain": "malawilii.org",
+        },
+        "namiblii": {
+            "country_code": "na",
+            "domain": "namiblii.org",
+        },
+        "senlii": {
+            "country_code": "sn",
+            "domain": "senlii.org",
+        },
+        "seylii": {
+            "country_code": "sc",
+            "domain": "new.seylii.org",
+        },
+        "sierralii": {
+            "country_code": "sl",
+            "domain": "new.sierralii.org",
+        },
+        "tanzlii": {
+            "country_code": "tz",
+            "domain": "new.tanzlii.org",
+        },
+        "tcilii": {
+            "country_code": "tc",
+            "domain": "new.tcilii.org",
+        },
+        "ulii": {
+            "country_code": "ug",
+            "domain": "new.ulii.org",
+        },
+        "zambialii": {
+            "country_code": "zm",
+            "domain": "new.zambialii.org",
+        },
+        "zanzibarlii": {
+            "place_code": "tz-znz",
+            "domain": "zanzibarlii.org",
+        },
+        "zimlii": {
+            "country_code": "zw",
+            "domain": "zimlii.org",
+        },
+    }
+
+    def __init__(self, app_name):
+        self.current_authority = self.RESOLVER_MAPPINGS[app_name.lower()]
+
+    def get_url_for_frbr_uri(self, parsed_uri):
```

Comment:
Minor: this method name implies it will return a URL but it actually returns a domain, and `get_best_authority` implies that method will return an authority but actually returns a domain.

Some variable and method renaming would make things more consistent.

Reply:
lol, always happy to renaime variables

Reply:
done